src/compressor/Compressor.h

```diff
@@ -16,7 +16,10 @@
 #define CEPH_COMPRESSOR_H
 
 #include "include/int_types.h"
-#include "include/Context.h"
+#include "include/memory.h"
+#include "include/buffer.h"
```

Comment:
@tchaikov since u know SnappyCompressor extends Compressor implemented in SnappyCompressor.h header file totally, so I think directly including "include/buffer.h" is ok.